style/js/shield_defs.js

```diff
@@ -2158,6 +2158,57 @@ export function loadShields(shieldImages) {
     null
   );
 
+  // Sweden
+  shields["SE:RV"] = roundedRectShield(
+    Color.shields.blue,
+    Color.shields.white,
+    Color.shields.white,
+    4,
```

Comment:
We've standardized on a corner radius of 2 pixels. If these shields are unrecognizable without a 4px radius, that is okay, but otherwise it should be changed to 2.

Reply:
If 2 is the default I can roll back to that, I just thought 4 pixels looked better as 2 pixels makes it almost look like there's no rounded corners at all.

Reply:
Changed to default